erc20_moc/Bet.noblock.js

```diff
@@ -37,7 +37,7 @@ class Bet {
   constructor(betId, outcomes) {
     this.betId = betId;
     this.fee = 0.01;
-    this.walletId = WALLET_PREFIX + betId;
+    this.walletId = betId;
```

Comment:
for backward compatibility please do not remove WALLET_PREFIX, we'll do it later. not everything at once :)